src/Language/PureScript/Docs/Render.hs

```diff
@@ -196,16 +201,16 @@ renderDeclaration (P.TypeClassDeclaration name args implies ds) title = do
 
   children = map renderClassMember ds
 
-  -- TODO: Comments for type class members
+  -- TODO: Comments for type class members?
```

Comment:
Ok - I've removed this. I also removed a similar one about data constructors having their own documentation.